### ISSUE TITLE
[Feature Request] Obfuscating some social actions

### ISSUE THREAD

**Issue by SansCipher** [lensvol/fl-oneclick-wiki#6]
Type: Feature Request
Title: Obfuscating some social actions
Explanation: Some social actions (for example "Allow an Acquaintance to Alleviate your Fears") exist in two exemplars, "dupe" and "sincere". I request that the exact version of such actions wouldn't be revealed via this extension.
Steps to Reproduce Current Behavior:
1) Have Player 1 use "Assist your friend with their Nightmares" actions on Player 2
2) Have Player 2 look at the resulting storylet and press the Wiki button provided by the extension
3) Observe that the button redirects the player to "Allow an Acquaintance to Alleviate your Fears (sincere)"
4) Have Player 1 use "Make scientific study of your friend's Nightmares" actions on Player 2
5) Have Player 2 look at the resulting storylet and press the Wiki button provided by the extension
6) Observe that the button redirects the player to "Allow an Acquaintance to Alleviate your Fears (dupe)"
Requested behavior:
In both cases, the wiki link opened should instead be "Allow an Acquaintance to Alleviate your Fears" (disambiguation page)

**Comment by lensvol** [comment]
Thank you for reporting this! It was totally unintended, but still kinda cool. :)

Will fix it as soon as I understand how the itself handles this situation.

**Comment by Saklad5** [comment]
I disagree: the information is being disclosed by Fallen London itself, and it is FBG’s job to fix it. Changing the extension will not prevent someone from distinguishing those actions anyway.

**Comment by lensvol** [comment]
@Saklad5 That makes sense, actually. And I could not see any special handling on FL UI as well, so works as intended?
